### PR TITLE
Additional clarification for setting up a validator/full node

### PR DESCRIPTION
**Pull request by mschmookler**
While attempting to set up a validator node, I ran into a few issues caused by some out of date docs. I'd like to make this clearer for others in the future.

### CODE REVIEW

**Comment by aw126** [comment]
Looks good to me, better to double-check the exact format of `pubkey`.

File: docs/getting-started/mainnet_validator.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ confirm transaction before signing and broadcasting [y/N]: y
   ::: tip You will be required to insert the following:
   - `--from`: The `cro...` address or the key name that holds your funds for initial delegation;
   - `--amount`: The amount of self-delegation provided to the validator as an initial staking;
-  - `--pubkey`: The validator public key ( See Step [3-2](#step-3-2-obtain-the-validator-public-key) above ) with **crocnclconspub** as the prefix;
+  - `--pubkey`: The validator public key. Can be found using `./chain-maind status`.;
```
Comment (lezzokafka) [inline docs/getting-started/mainnet_validator.md:289]:
```suggestion
  - `--pubkey`: The validator public key. Can be found using `./chain-maind tendermint show-validator`.;
```

**Review by lezzokafka — APPROVED** [review]
Thanks @mschmookler for your contribution! 👍

**Pull request merged**